src/MAUI/Maui.Samples/Samples/Search/FindAddress/FindAddress.xaml.cs

```diff
@@ -139,7 +139,7 @@ private async Task<Graphic> GraphicForPoint(MapPoint point)
             // Get image as a stream from the resources
             // Picture is defined as EmbeddedResource and DoNotCopy
             Stream resourceStream = currentAssembly.GetManifestResourceStream(
-                "ArcGISRuntimeMaui.Resources.PictureMarkerSymbols.pin_star_blue.png");
+                "ArcGISMapsSDK.Resources.PictureMarkerSymbols.pin_star_blue.png");
```

Comment:
We should avoid using `ArcGISMapsSDK` here. Perhaps need to think of a better name.